lib/login.js

```diff
@@ -3,61 +3,120 @@ var client = new RegClient({});
 var fs = require('fs');
 var path = require('path');
 
-module.exports = function (npmUser, npmPass, npmEmail, npmRegistry, npmScope, quotes, configPath) {
-    var registry = npmRegistry || 'https://registry.npmjs.org';
-    var scope = npmScope;
-
-    client.adduser(registry, {
-        auth: {
-            username: npmUser,
-            password: npmPass,
-            email: npmEmail
+// jshint freeze:false
+if (!Array.prototype.findIndex) {
```

Comment:
Next time around, you can use Array.indexOf, so you don't have to do all this Polyfill etc stuff.